examples/st2110-22.sdp

```diff
@@ -1,12 +1,12 @@
 v=0
-o=- 1543226715 1543226715 IN IP4 172.29.80.65
-s=Demo Video Stream
+o=- 1443716955 1443716955 IN IP4 192.168.1.2
+s=SMPTE ST2110-22 JPEG XS
 t=0 0
-m=video 27346 RTP/AVP 97
-c=IN IP4 232.80.177.113/32
-b=AS:216000
-a=source-filter: incl IN IP4 232.80.177.113 172.29.80.65
-a=ts-refclk:ptp=IEEE1588-2008:EC-46-70-FF-FE-00-CE-DE:0
-a=rtpmap:97 vc2/90000
-a=fmtp:97 profile=HQ; width=1920; height=1080; TP=2110TPW; exactframerate=25
-a=mediaclk:direct=0 rate=90000
+m=video 30000 RTP/AVP 112
+c=IN IP4 224.1.1.1/64
+b=AS:116000
+a=source-filter: incl IN IP4 224.1.1.1 192.168.1.2
+a=rtpmap:112 jxsv/90000
+a=fmtp:112 packetmode=0; profile=High444.12; level=1k-1; sublevel=Sublev3bpp; sampling=YCbCr-4:2:2; width=1280; height=720; exactframerate=60000/1001; depth=10; colorimetry=BT709; TCS=SDR; RANGE=FULL; SSN=ST2110-22:2019; TP=2110TPN; 
```

Comment:
@garethsb This was taken from bcp-006-01 example and fails the new sdpoker tests since TP according to the specs I have should be either 2110TPNL or 2110TPW.  I've added an issue also to the bcp-006-01 repo.

Reply:
There was an oversight in ST 2110-22:2019, corrected in ST 2110-22:2022, as there is a subtle difference between 2110TPN and 2110TPNL which is still relevant to ST 2110-22.